doc/rst/opt_adat_rx.rst

```diff
@@ -0,0 +1,72 @@
+|newpage|
+
+ADAT Receive
+============
+
+The codebase supports a single ADAT receiver that can receive up to eight channels of audio at a sample rate
+of 44.1kHz or 48kHz over an optical interface.
+In order to provide ADAT functionality ``lib_xua`` uses `lib_adat <https://www.github.com/xmos/lib_spdif>`_.
+
+Basic configuration of ADAT receive functionality is achieved with the following :ref:`defines<opt_adat_rx_defines>`:
+
+.. _opt_adat_rx_defines:
+
+|beginfullwidth|
+
+.. list-table:: ADAT RX defines
+   :header-rows: 1
+   :widths: 25 40 40
+
+   * - Define
+     - Description
+     - Default
+   * - ``XUA_ADAT_RX_EN``
+     - Enable ADAT receive
+     - ``0`` (Disabled)
+   * - ``ADAT_RX_MAX_CHANS``
+     - Maximum number of channels to receive over the ADAT interface
+     - ``8, 4 or 2`` depending on the min and max sampling freq supported in the application
+   * - ``ADAT_RX_INDEX``
+     - Start channel index of ADAT RX channels
+     - N/A (must be defined by the application)
+
+|endfullwidth|
+
+.. note::
```

Comment:
Remove this note as it applies to tx only.

Reply:
done